### PR TITLE
[TECHNICAL-SUPPORT] LPS-77038

### PR DESCRIPTION
**Pull request by jonathanmccann**
/cc @ChrisKian

### CODE REVIEW

**Comment by jonathanmccann** [comment]
ci:test

File: portal-impl/src/com/liferay/portal/util/PortalImpl.java [hunk 1]
```diff
@@ -8640,6 +8640,10 @@ private long _getScopeGroupId(
 		if (sitesContentSharingWithChildrenEnabled !=
 				Sites.CONTENT_SHARING_WITH_CHILDREN_DISABLED) {
 
+			if (!curGroup.isRoot()) {
```
Comment (jonathanmccann) [inline portal-impl/src/com/liferay/portal/util/PortalImpl.java]: @ChrisKian I just noticed this, but curGroup isn't guaranteed to be non-null, is it? Might need to throw in a null check here.

Comment (ChrisKian) [reply portal-impl/src/com/liferay/portal/util/PortalImpl.java]: You're exactly right, curGroup could be null

**Comment by ealonso** [comment]
Hey @jonathanmccann @ChrisKian,

Could you please fix the possible null pointer and send it to core infrastructure team? I think those changes in PortalImpl should be reviewed by someone in this team, for asset browser perspective everything is correct.

Also there is a source formatter error.

Regards,
Eudaldo.

**Comment by ChrisKian** [comment]
Resent, see https://github.com/jonathanmccann/liferay-portal/pull/1503

**Pull request closed without merging**